features-json/css-scrollbar.json

```diff
@@ -1,13 +1,17 @@
 {
   "title":"CSS scrollbar styling",
-  "description":"Non-standard methods of styling scrollbars.",
-  "spec":"https://webkit.org/blog/363/styling-scrollbars/",
+  "description":"Methods of styling scrollbars' color and width.",
+  "spec":"https://drafts.csswg.org/css-scrollbars-1/",
   "status":"unoff",
   "links":[
     {
-      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=77790",
+      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=1460109",
       "title":"Firefox support bug"
     },
+    {
+        "url":"https://webkit.org/blog/363/styling-scrollbars/",
+        "title":"Webkit blog post describing their non-standard support"
+    },
```

Comment:
And remove the comma as well

Reply:
I genuinely fail to see how this comma is extra.

Reply:
Please apply the fix. Until you do that, it is considered error.